### PR TITLE
Circuit component descriptions and module names are now visible to the naked eye.

### PR DESCRIPTION
**Pull request by Gurkenglas**
## About The Pull Request

`display_desc` is now `desc`. `module/set_display_name` now updates `name`.

## Why It's Good For The Game

Unified purposes are good. In particular, inventory management is easier if my different modules have different names.

## Changelog
:cl:
qol: Circuit component descriptions and module names are now visible to the naked eye.
/:cl:

### CODE REVIEW

**Comment by Watermelon914** [comment]
They're display name because that's what they're displayed in the ui.
Imo, the current name should not change, that should still be display_name.
I don't mind desc being visible for all, but display_name and the actual name should definitely be different because all capitalized items look bad.

**Comment by Gurkenglas** [comment]
I accidentally originally branched off of another PR, fixed and force-pushed.

File: code/modules/explorer_drone/exploration_site.dm [hunk 1]
```diff
@@ -150,7 +150,7 @@ GLOBAL_LIST_EMPTY(exploration_sites)
 /datum/exploration_site/proc/display_name()
 	return revealed ? name : "Anomaly"
 
-/datum/exploration_site/proc/display_description()
+/datum/exploration_site/proc/description()
```
Comment (Mothblocks) [inline code/modules/explorer_drone/exploration_site.dm:153]: what

File: code/modules/explorer_drone/exploration_site.dm [hunk 2]
```diff
@@ -167,7 +167,7 @@ GLOBAL_LIST_EMPTY(exploration_sites)
 	.["ref"] = ref(src)
 	.["name"] = display_name()
 	.["coordinates"] = coordinates
-	.["description"] = display_description()
+	.["description"] = description()
```
Comment (Mothblocks) [inline code/modules/explorer_drone/exploration_site.dm:170]: what

File: code/modules/wiremod/integrated_circuit.dm [hunk 3]
```diff
@@ -484,6 +485,10 @@
 /obj/item/integrated_circuit/proc/set_display_name(new_name)
 	display_name = new_name
 
+/obj/item/integrated_circuit/module/set_display_name(new_name)
+	..()
+	name = "module ([new_name])"
```
Comment (Mothblocks) [inline code/modules/wiremod/integrated_circuit.dm:490]: what?

File: code/modules/explorer_drone/exploration_site.dm (hunk 2 above)
Comment (Gurkenglas) [reply code/modules/explorer_drone/exploration_site.dm:170]: yeah, uh, unfortunate, good catch.

File: code/modules/wiremod/integrated_circuit.dm (hunk 3 above)
Comment (Gurkenglas) [reply code/modules/wiremod/integrated_circuit.dm:490]: Ah, i misunderstood some advice I got on how to rename only the modules. Should have tested that again.

**Comment by Watermelon914** [comment]
lgtm

**Pull request merged**